src/main/java/com/minecolonies/coremod/network/messages/GuardTaskMessage.java

```diff
@@ -39,7 +39,13 @@ public GuardTaskMessage()
      * @param building       View of the building to read data from.
      * @param assignmentMode assignmentMode of the particular farmer.
      */
-    public GuardTaskMessage(@NotNull final BuildingGuardTower.View building, final int job, final boolean assignmentMode, final boolean patrollingMode, final boolean retrieval, final int task )
+    public GuardTaskMessage(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): job, patrollingMode, retrieval, task [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)